src/python/grpcio/grpc/_service_reflection.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2020, the gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import grpc
+
+from google.protobuf import descriptor_pb2
+from google.protobuf import symbol_database
+
+DESCRIPTOR_KEY = 'DESCRIPTOR'
+
+
+def get_servicer_class_name(service_descriptor):
+    return service_descriptor.name + "Servicer"
+
+
+class ReflectiveServicerType(type):
+    """Metaclass for Servicer classes created at runtime from ServiceDescriptors
+    """
+
+    def __init__(cls, unused_name, unused_bases, namespace):
+        if DESCRIPTOR_KEY not in namespace:
+            return
+
+        service_descriptor = namespace[DESCRIPTOR_KEY]
+
+        def _unimplemented_method(self, request, context):
+            context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+            context.set_details('Method not implemented!')
+            raise NotImplementedError('Method not implemented!')
```

Comment:
optional: Does `context.abort(grpc.StatusCode.UNIMPLEMENTED, 'Method not implemented!')` looks better?

Reply:
Perhaps. But that's not what we currently have in our generated code. If we change that here, we should change our generated code as well.